var/spack/repos/builtin/packages/ffte/package.py

```diff
@@ -9,7 +9,9 @@
 
 
 class Ffte(Package):
-    'Fastest Fourier Trnasform in the East.'
+    'Fastest Fourier Transform in the East. Provides FFT for powers of 2, 3,
```

Comment:
A check failed on that, we can't merge without this fixed:
SyntaxError: invalid syntax in /home/spack-test/spack/var/spack/repos/builtin/packages/ffte/package.py, line 13

Use `"""` instead of `'` for the class and method documentation strings (like before).
Running flake8 as style check on your change should be enough the syntax after changing to `"""`.